Multiplatform/Localizable.xcstrings

```diff
@@ -210,6 +282,7 @@
       }
     },
     "%lld episodes.unplayed" : {
+      "comment" : "Could differ depending on if there's one unplayed or many unplayed.",
```

Comment:
Good idea, could you do it and add the translation?

Reply:
Should be fixed now.